examples/arm/setup.sh

```diff
@@ -219,8 +224,9 @@ function setup_tosa_reference_model() {
     # errors for the early codebase
     cd "${root_dir}"
     if [[ ! -e reference_model ]]; then
-        git clone https://review.mlplatform.org/tosa/reference_model -b v0.80
+        git clone ${tosa_reference_model_url}
         cd reference_model
+        git reset --hard ${tosa_reference_model_rev}
```

Comment:
`git reset` will change what the default branch points to; maybe just
```suggestion
        git checkout ${tosa_reference_model_rev}
```

But the reset will work.

Reply:
Just the detached head warning I don't like with check out, but no strong preference, will update.

Reply:
Reset is fine if you'd rather do that. Stack overflow seems to do either of them. And since we're not making changes to the repo it's not a big deal -- `origin/main` won't change even with a reset